### DISCUSSION TITLE
How to perform a logout?

### DISCUSSION THREAD

**Discussion by riazza** [panva/oauth4webapi#75] [Q&A] [answered]
Hi!

I am wondering how to implement a "logout" for a logged-in user? Is there any convenience function I can use? 

Thanks a lot for your help!

**Comment by panva** [comment, marked as answer]
Since this module does not deal with your app's sessions it cannot deal with your session's termination. 

For AS logout please have a look at the OIDC RP-Initiated Logout which from the module's perspective is a no-op since it's just about forming a redirect link to the AS end session endpoint, not unlike forming the authorization endpoint url with parameters.
